### PR TITLE
Fix buildbot

### PR DESCRIPTION
**Pull request by mugling**
Fallout from  #18541 and #18485

To avoid recurrence before each merge:
- Run `--check-mods`
- Wait for buildbot to verify JSON (green tick symbol) or manually run `make lint`

Self-merge as it fixes buildbot issues currently being reported in other PR's

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
Why exactly is the build bot having these problems now? I was under the impression that lint was supposed to improve the code in some nebulous fashion, not cause the auto-builder's basic functionality to break.

**Comment by mugling** [comment]
The first breakage last week was a syntax error merged in #18244. That doesn't have anything to do with linting. It was reported in #18423 and fixed in #18424. It is likely the original PR was merged untested as the error is apparent immediately on load. Testing the PR and/or running `--check-mods` before merging would prevent future recurrences.

The recent breakage is probably caused by #18541 and was the result of lint regressions. It appears the PR was merged relatively soon after being opened and before automated checks had completed (green tick symbol). The linter cannot protect against regressions if a branch is merged before the automated checks have run. To avoid wait for the green tick symbol to appear next to the commit.

On occasion these checks can be delayed. A similar regression was introduced by #18485 and could be contributory to the above.

Which brings us back to here:

> To avoid recurrence before each merge:
> 
>     Run --check-mods
>     Wait for buildbot to verify JSON (green tick symbol) or manually run make lint

**Comment by mugling** [comment]
tl;dr - last week we merged a PR without testing which contained an error and this week we didn't wait for the buildbot to verify PR's before merging

**Comment by mugling** [comment]
Breakage last night was `astyle` regression. The fix (f80fe5e) looks problematic.
